pages/api/updateAlgolia.ts

```diff
@@ -0,0 +1,15 @@
+import path from 'path';
+import fs from 'fs';
+import url from 'url';
+import { updateIndex } from '@/lib/algolia/getRecords';
+
+export default async function handler(_req, res) {
+    const jsonDirectory = path.join(process.cwd());
+    // @ts-ignore
+    const dummyVar = fs.readdirSync(`${jsonDirectory}/common`);
+    // @ts-ignore
+    const dummyLink = url.pathToFileURL(path.resolve(`${jsonDirectory}/common`));

```

Comment:
this is not used. why do we need it ?

Reply:
fs and url are required for the function call, will not be present after bundling otherwise. Used to avoid unused import errors